substrate/frame/referenda/src/lib.rs

```diff
@@ -117,27 +117,6 @@ pub mod benchmarking;
 
 pub use frame_support::traits::Get;
 
-#[macro_export]
-macro_rules! impl_tracksinfo_get {
```

Comment:
Thanks for having done it, to avoid breaking change we could also keep the macro but make it deprecated. The deprecation message can say: tracks info no longer requires get, the macro invocation can be removed.

But keep it removed is also ok to me.

Reply:
After a small evaluation, I concluded that removing it is better since we're eliminating some potentially unused code at the almost negligible cost of deleting 5 calls to this macro in the `runtimes` repo (plus one or two for every runtime being built outside the scope of Polkadot).

To me keeping it removed it's a no brainer.

Reply:
I had to get full sherlock homes to find out whether its okay to delete this macro invocation or whether it had to be replaces. Just trying doesn't help cause you are always at 300+ errors when upgrading your runtime.

Reply:
Yes, you we can reintroduce it an mark it as deprecated.